dev/firely/docker-compose.yaml

```
@@ -0,0 +1,12 @@
+version: '3'
+
+services:
+  firely:
+    image: firely/server:${FHIR_IMAGE_TAG:-5.2.0}
+    environment:
+        VONK_Administration:Repository: SQLite
+        VONK_Administration:SQLiteDbOptions:ConnectionString: "Data Source=./data/vonkadmin.db"
+        VONK_Administration:SQLiteDbOptions:AutoUpdateDatabase: "true"
```

Fixup indent level (2 vs 4 spaces)
```suggestion
      VONK_Administration:Repository: SQLite
      VONK_Administration:SQLiteDbOptions:ConnectionString: "Data Source=./data/vonkadmin.db"
      VONK_Administration:SQLiteDbOptions:AutoUpdateDatabase: "true"
```